packages/devtools_app/lib/src/url_strategy/web.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:flutter_web_plugins/flutter_web_plugins.dart';
```

Comment:
move the url_strategy folder under src/config_specific

Reply:
Done!